src/OAuth/OAuth1/Signature/Signature.php

```diff
@@ -54,7 +54,7 @@ public function getSignature(UriInterface $uri, array $params, $method = 'POST')
         parse_str($uri->getQuery(), $queryStringData);
 
         foreach (array_merge($queryStringData, $params) as $key => $value) {
-            $signatureData[rawurlencode($key)] = rawurlencode($value);
+            $signatureData[rawurlencode($key)] = $value ? rawurlencode($value): $value;
```

Comment:
If `$value` is null, then this will set the value of `$signatureData[rawurlencode($key)]` to null, but it needs to be an empty string. Something like this might be better:

```php
$signatureData[rawurlencode($key)] = $value ? rawurlencode($value) : '';
```

Reply:
can you make that PR?  Or submit a new one altogether?

Reply:
Why would it be a separate PR? The issue is being introduced in this PR, and so normally I'd imagine that a new commit would be added here to sort out this sort of thing. Or am I misunderstanding?

Reply:
it's mostly because I'm not 100% sure how to tweak my PR!  

I managed to burn through all my Sentry errors in 24 hours because I can't figure out how to turn off deprecations, and I was doing a batch upload.  Sigh.

Reply:
Oh no worries then, yep I can make one. :-) Will try to later today.

@carlos-mg89 Are you around to review PRs?

Reply:
I've had a go in #15; see what you think.

Reply:
I've merged already the #15 PR. So I presume this one makes no sense anymore.

Thanks a lot to both of you for helping to make this fork more useful, secure and effective.